chapters/inheritance.tex

```diff
@@ -252,18 +242,15 @@ \section{Modifications}\label{modifications}
 There are three kinds of constructs in the Modelica language in which modifications can occur:
 \begin{itemize}
 \item
-  Variable declarations.
+  Variable declarations
```

Comment:
I don't like that list items have different capitalization, and since they are no longer individual sentences I don't see a need for a capital letter.
```suggestion
  variable declarations
```